### ISSUE TITLE
Connection not private in Chrome when clicking on https://ok-b.org/

### ISSUE THREAD

**Issue by mhulse** [ok-borg/borg#27]
Just thought I would mention (using latest Chrome on MacOS):

![screenshot 2016-10-14 09 34 03](https://cloud.githubusercontent.com/assets/218624/19395384/37106b22-91f2-11e6-8e76-2b51d26b767f.png)

**Comment by crufter** [comment]
@mhulse, thank you very much for reporting this.

We have to investigate this, since I don't see it. What's your version of chrome?

Edit: Just realised you said latest.

**Comment by mhulse** [comment]
Hi @crufter, thanks for the quick reply. Sorry that I did not provide this information in my first post:

Version 53.0.2785.143 (64-bit)

As I went to view the version, Chrome updated. I'm restarting Chrome now and will see if that helped.

Maybe this is just a problem with my computer/browser? I would hate to waste anyone's time here.

**Comment by crufter** [comment]
@mhulse You are not wasting our time, we appreciate feedback. I don't think the problem is on your end, if other encrypted sites work for you, this should as well.

I upgraded to the latest chrome and I can't see the issue, I have seen some alerts due to cached html where we loaded jquery from a 'http' source, but this is something else I believe.

I will try my best to reproduce this. Will come back to you, thanks.

Edit: tested in Safari as well and it works. The certificate is pretty new, acquired with Let's Encrypt, but I think it should work flawlessly immediately.
